eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -20,6 +45,11 @@ function Get-java-DocsMsTocData($packageMetadata, $docRepoLocation) {
         $packageLevelReadmeName = $readmeMetadata.DocsMsReadMeName
     }
 
+    return $packageLevelReadmeName
+}
+
+function Get-java-DocsMsTocData($packageMetadata, $docRepoLocation) {
+    $packageLevelReadmeName = Get-java-PackageLevelReadme -packageMetadata $packageMetadata
```

Comment:
```suggestion
    $packageLevelReadmeName = &$GetPackageLevelReadmeFn -packageMetadata $packageMetadata
```

Is `Get-java-PackageLevelReadme` defined somewhere else?

Reply:
Talked offline. Will use common helper function for the implementation.